src/reference/asciidoc/whats-new.adoc

```diff
@@ -16,3 +16,9 @@ When using <<receiving-batch,batching>>, `@RabbitListener` methods can now recei
 
 The Log4J and Logback `AmqpAppender` s now support a `verifyHostname` SSL option.
 See <<logging>> for more information.
+
+===== MessageListenerAdapter Changes
+
+`MessageListenerAdapter` provides now a new method to build an array of arguments to be passed into target listener
+and an old one is deprecated.
```

Comment:
Would be better to point at least method name.

Imaging the situation when I already know the Framework and their is just enough for me to read `What's New` to start using new features. I don't see reason to send me to the big chapter if by the method name I would know what to look for in the source code.